src/components/ContentHighlights/HighlightStepper/ContentConfirmContentCard.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Delete } from '@edx/paragon/icons';
+import { IconButton, Icon } from '@edx/paragon';
+import { connect } from 'react-redux';
+import ContentHighlightCardItem from '../ContentHighlightCardItem';
+import { useContentHighlightsContext } from '../data/hooks';
+import { generateAboutPageUrl } from '../data/constants';
+
+const ContentConfirmContentCard = ({ enterpriseSlug, original }) => {
+  const { deleteSelectedRowId } = useContentHighlightsContext();
+  const [deleteKey, setDeleteKey] = useState(null);
+  const [isDeleted, setIsDeleted] = useState(false);
+  const {
+    title,
+    contentType,
+    partners,
+    cardImageUrl,
+    originalImageUrl,
+    firstEnrollablePaidSeatPrice,
+    aggregationKey,
+    key,
+  } = original;
+  // eslint-disable-next-line no-shadow
+  useEffect(() => {
+    if (deleteKey === aggregationKey) {
+      deleteSelectedRowId(deleteKey);
+      setIsDeleted(true);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [deleteKey, aggregationKey]);
+  if (isDeleted) {
+    return (
+      null
+    );
```

Comment:
[curious] Might need some more clarity on what's happening here. When the selected row id is removed via `deleteSelectedRowId`, wouldn't the `algoliaFilters` passed to `<Configure />` get re-computed to not include the now-deleted content item, where this card would no longer be rendered regardless of the internal `isDeleted` state?

Reply:
Running through it without this code section it seems to be working. At some point I had floating cards with the container remaining without the cards reordering themselves filling in the empty gaps.